docs/api/standard_library_types.md

```diff
@@ -133,7 +133,8 @@ print(m.model_dump())
     * `timedelta`; an existing `timedelta` object
     * `int` or `float`; assumed to be seconds
     * `str`; the following formats are accepted:
-        * `[-][DD ][HH:MM]SS[.ffffff]`
+        * `[-][DD]D[,][HH:MM]SS[.ffffff]`
```

Comment:
```suggestion
        * `[-][DD]D[,][HH:MM:]SS[.ffffff]`
```
?

Reply:
Maybe I'm just not familiar with whatever syntax is used to represent a "format" here